packages/react-devtools-shared/src/backend/views/Highlighter/Overlay.js

```diff
@@ -230,22 +233,10 @@ export default class Overlay {
       name = elements[0].nodeName.toLowerCase();
 
       const node = elements[0];
-      const hook: DevToolsHook =
-        node.ownerDocument.defaultView.__REACT_DEVTOOLS_GLOBAL_HOOK__;
-      if (hook != null && hook.rendererInterfaces != null) {
-        let ownerName = null;
-        // eslint-disable-next-line no-for-of-loops/no-for-of-loops
-        for (const rendererInterface of hook.rendererInterfaces.values()) {
-          const id = rendererInterface.getFiberIDForNative(node, true);
-          if (id !== null) {
-            ownerName = rendererInterface.getDisplayNameForFiberID(id, true);
-            break;
-          }
-        }
-
-        if (ownerName) {
-          name += ' (in ' + ownerName + ')';
-        }
+      const ownerName = this.agent.getDisplayNameForNode(node);
```

Comment:
Maybe instead of making `getDisplayNameForNode` you can use `getBestMatchingRendererInterface`, `getFiberIDForNative`, and `gwetDisplayNameForFiberID`?

Reply:
Thank you guys! The company have a phone number?